dusty/commands/utils.py

```diff
@@ -20,3 +23,29 @@ def exec_docker(*args):
     updated_env.update(get_docker_env())
     args += (updated_env,)
     os.execle(_executable_path('docker'), 'docker', *args)
+
+def pty_fork(*args):
```

Comment:
wondering if this should go in the file with: check_and_log_output_and_error_demoted instead of this file.